tests/AssertTest.php

```diff
@@ -287,7 +288,8 @@ public function getTests()
             array('uuid', array('ff6f8cb0-c57da-51e1-9b21-0800200c9a66'), false),
             array('uuid', array('af6f8cb-c57d-11e1-9b21-0800200c9a66'), false),
             array('uuid', array('3f6f8cb0-c57d-11e1-9b21-0800200c9a6'), false),
-
+            array('throws', array(function() { throw new LogicException('test'); }, 'LogicException'), true),
+            array('throws', array(function() { throw new LogicException('test'); }, 'IllogicException'), false),
```

Comment:
If the check method would be with `Closure` type in method prototype then it won't needed.

so only my 2nd point would be valid